build/gen_stub.php

```diff
@@ -453,7 +474,7 @@ public function equals(ReturnInfo $other): bool {
     }
 }
 
-class FuncInfo {
+class FuncInfo extends MemberInfo {
```

Comment:
I'm not sure `MemberInfo` is the best term

Reply:
Is a common base class even useful? I would expect them to be essentially disjoint.

Reply:
It's not essential, I guess. But then is it ok if I extract the `getFlagsAsString()` method to an utility class? Or should I just copy-paste?

Reply:
Ok, problem solved (not yet pushed). I went with two separate implementations (actually it's already 3, since there's one for class flags). And yes, the supported flags are really different.